apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -43,6 +43,25 @@ export function ListBoxInline({ app, fieldIdentifier, stateName = '$', options =
     update = undefined,
     dense = false,
   } = options;
+  let { frequencyMode } = options;
+
+  switch (true) {
+    case ['none', 'N', 'NX_FREQUENCY_NONE'].includes(frequencyMode):
+      frequencyMode = 'N';
+      break;
+    case ['value', 'V', 'NX_FREQUENCY_VALUE', 'default'].includes(frequencyMode):
+      frequencyMode = 'V';
+      break;
+    case ['percent', 'P', 'NX_FREQUENCY_PERCENT'].includes(frequencyMode):
+      frequencyMode = 'P';
+      break;
+    case ['relative', 'R', 'NX_FREQUENCY_RELATIVE'].includes(frequencyMode):
+      frequencyMode = 'R';
+      break;
+    default:
+      frequencyMode = 'N';
+      break;
```

Comment:
Interesting way of using a switch-statement, looks quite clean 😄  Do we need this last `break` though or is it a best practice?

Reply:
Seems like best-practice according to some since default may by accident move upwards, above a `case`.